doc/rose-rug-advanced-tutorials-parameterization-and-jinja2.html

```diff
@@ -0,0 +1,294 @@
+          <a href="http://www.metoffice.gov.uk">Met Office</a>. See
+          <a href="rose-terms-of-use.html">Terms of Use</a>.<br />
+          This document is released under the <a href=
+          "http://www.nationalarchives.gov.uk/doc/open-government-licence/"
+          rel="license">Open Government
+          Licence</a>.</span></span></li>
+
+          <li><span id="rose-version" class=
+          "navbar-text"></span></li>
+        </ul>
+      </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+  </nav>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div id="panel-main" class="panel panel-default">
+          <div class="panel-heading">
+            <h1 class="slides-only logo logo-large">Parameterization and
+            Jinja2</h1>
+            <h1 class="outline-only">Rose Advanced Tutorial: Parameterization
+            and Jinja2</h1>
+          </div>
+
+          <div id="body-main" class="panel-body presentation">
+            <div class="handout" id="content"></div>
+
+            <div class="slide slide-title">
+              <span>Parameterization and Jinja2</span>
+            </div>
+
+            <div class="slide">
+              <h2 id="intro">Introduction</h2>
+
+              <p>This tutorial walks you through combining Jinja2 and task
+              parameterization in your suites. It assumes you are already
+              familiar with using Jinja2. If not, see the
+              <a href="rose-rug-advanced-tutorials-jinja2.html">Jinja2
+              tutorial</a>.</p>
+
+              <p>By combining Jinja2 with task parameterization you can make
+              use of further flexibility in generating your
+              <var>suite.rc</var>.</p>
+
+            </div>
+
+            <div class="slide">
+              <h2 id="purpose">Purpose</h2>
+
+              <p>The purpose of this tutorial is to demonstrate some of the
+              flexibility for suite design you can obtain by combining Jinja2
+              with parameterization. Using parameterization allows you to
+              minimise the amount of graphing in your suite while using Jinja2
+              allows you some further flexibility and configurability on top of
+              that.</p>
+
+            </div>
+
+            <div class="slide">
+              <h2 id="example">Example</h2>
+              <p>Here, we'll build on the "hello world" example suite from the
+              <a href="rose-rug-suites-I.html#parameterization">
+              parameterization</a> section of the Suites I part of the user
+              guide.</p>
+
+              <p>To start with, we will extend the suite so that a user can
+              pass through a set of worlds to say hello to via an entry in the
+              <var>rose-suite.conf</var> file.</p>
+
+            </div>
+
+            <div class="slide">
+              <h3 id="example-2">Example (2)</h3>
```

Comment:
I think these headings should only appear in sideshow mode, to do so add the class `slides-only` and remove the id.